packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -0,0 +1,296 @@
+"""
+    Models used in storage API
+
+    IMPORTANT: do NOT couple these schemas until properly
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+from uuid import UUID
+
+from pydantic import BaseModel, Field, constr
+
+from .basic_regex import UUID_RE
+
+
+# /
+class HealthCheck(BaseModel):
+    name: Optional[str] = None
+    status: Optional[str] = None
+    api_version: Optional[str] = None
+    version: Optional[str] = None
```

Comment:
careful with this: do you expect the jsonschema to allow for null? then add the exporter we saw the other day.

Reply:
True. Actually they are not nullable. Done